src/Kusto/examples/New-AzKustoDataConnection.md

```diff
@@ -1,9 +1,7 @@
 ### Example 1: Create a new EventHub data connection
 ```powershell
-New-AzKustoDataConnection -ResourceGroupName "testrg" -ClusterName "testnewkustocluster" -DatabaseName "mykustodatabase" -DataConnectionName "myeventhubdc" -Location "East US" -Kind "EventHub" -EventHubResourceId "/subscriptions/$subscriptionId/resourcegroups/testrg/providers/Microsoft.EventHub/namespaces/myeventhubns/eventhubs/myeventhub" -DataFormat "JSON" -ConsumerGroup '$Default' -Compression "None" -TableName "Events" -MappingRuleName "EventsMapping"
-```
+PS C:\> New-AzKustoDataConnection -ResourceGroupName "testrg" -ClusterName "testnewkustocluster" -DatabaseName "mykustodatabase" -DataConnectionName "myeventhubdc" -Location="East US" -Kind "EventHub" -EventHubResourceId "/subscriptions/$subscriptionId/resourcegroups/testrg/providers/Microsoft.EventHub/namespaces/myeventhubns/eventhubs/myeventhub" -DataFormat "JSON" -ConsumerGroup 'Default' -Compression "None" -TableName "Events" -MappingRuleName "EventsMapping"
```

Comment:
We recently fixed some issues in help documents, see https://github.com/Azure/azure-powershell/pull/17653 -- that's what caused the conflict. However it seems by resolving the conflicts the issues are back. For example `-Location="East US"` where `=` is invalid in PowerShell.
Also we intentionally split the input and output of each example in two code blocks. And we removed the `PS C:\>` prompt for it to be on-click runnable on docs.microsoft.com .

So please check those files with conflict again. I'm almost certain you can simply take the version on the `generation ` branch.